src/disco/tiles.h

```diff
@@ -112,4 +112,95 @@ typedef struct __attribute__((packed)) {
   uchar  last_entry_hash[32];
 } fd_poh_init_msg_t;
 
+/* A fd_txnm_t is a parsed meta transaction, containing not just the
+   payload */
+
+struct fd_txnm {
+   /* The computed slot that this transaction is referencing, aka. the
+      slot number of the reference_blockhash.  If it could not be
+      determined, this will be a slot around 150 in the future. */
+   ulong    reference_slot;
+
+   ushort   payload_sz;
+
+   /* Can be computed from the txn_t but it's expensive to parse again,
+      so we just store this redundantly. */
+   ulong    txn_t_sz;
+
+   /* An 8 byte tag of the first signature in the transaction, for use
+      by dedup. */
+
+   /* There are three additional fields at the end here, which are
+      variable length and not included in the size of this struct.
+   uchar                  payload[ ]
+   fd_txn_t               txn_t[ ]
+   fd_txn_acct_addr_lut_t alut[ ] */
```

Comment:
Do you mean the expanded address lookup tables? If so, that's not the right type. If you actually mean the field of type `fd_txn_acct_addr_lut_t`, I'd call that part of the `fd_txn_t`.

Reply:
Based on the footprint calculation, I think the type should be `fd_acct_addr_t`